lib/Runtime/Library/JavascriptLibrary.cpp

```diff
@@ -1357,6 +1403,26 @@ namespace Js
             DeferredTypeHandler<InitializeWeakSetConstructor>::GetDefaultInstance());
         AddFunction(globalObject, PropertyIds::WeakSet, weakSetConstructor);
 
+        registryConstructor = nullptr;
```

Comment:
nitpick: either we all do it pre-initialization, or all do it if () else() style. seems like we are more consistent in doing if...else style